main.py

```diff
@@ -1,6 +1,10 @@
+from environs import Env
 import telebot
 
 
+env = Env()
+env.read_env()
+env.str("SECRET_TOKEN") # => SOMETHING
 bot = telebot.TeleBot("secret_token", parse_mode='html')
```

Comment:
значение из .env нужно присвоить в переменую и передать в TeleBot

```suggestion
BOT_TOKEN = env.str("TELEGRAM_BOT_TOKEN")
bot = telebot.TeleBot(BOT_TOKEN, parse_mode='html')
```